tests/codex/blockexchange/engine/testengine.nim

```diff
@@ -142,6 +142,15 @@ asyncchecksuite "NetworkStore engine handlers":
     localStore: BlockStore
     blocks: seq[Block]
 
+  const NOP_SEND_WANT_LIST = proc(
```

Comment:
You probably want to use a more nimish style of constant naming for this `NopSendWantList`